feature/platform/ate_tests/telemetry_inventory_test/telemetry_inventory_test.go

```diff
@@ -588,10 +588,14 @@ func ValidateComponentState(t *testing.T, dut *ondatra.DUTDevice, cards []string
 		}
 
 		if p.operStatus != "" {
-			operStatus := gnmi.Get(t, dut, component.OperStatus().State()).String()
-			t.Logf("Hardware card %s OperStatus: %s", card, operStatus)
-			if operStatus != activeStatus {
-				t.Errorf("component.OperStatus().Get(t) for %q): got %v, want %v", card, operStatus, p.operStatus)
+			if deviations.FanOperStatusUnsupported(dut) && strings.Contains(card, "Fan") {
+				t.Logf("Skipping check for fan oper-status")
```

Comment:
please make skip message specific `t.Logf("Skipping check for fan oper-status due to deviation FanOperStatusUnsupported")`

Reply:
Done. Thanks!

Reply:
@mojiiba, could you please approve the changes. Thanks!